src/main/java/eu/eutampieri/catacombs/ui/World.java

```diff
@@ -17,20 +17,19 @@
 import java.util.stream.Stream;
 
 public final class World {
-    // private final BufferedImage background;
+    private final static int BOSS_SPAWN_RANGE = 10;
```

Comment:
imo too low maybe consider extending to at least 100

Reply:
will do

Reply:
Please check now and if it's ok please merge.